### PR TITLE
Doc fix, Any/Object

### PR DESCRIPTION
**Pull request by arsonik**

### CODE REVIEW

File: Sources/Core/AppEvents/AppEventsLogger.swift [hunk 1]
```diff
@@ -141,6 +141,18 @@ public class AppEventsLogger {
...
 
   //--------------------------------------
+  // MARK: - Push Notification
+  //--------------------------------------
+
+  /**
+   Sets a device token to register the current application installation for push notifications.
+   - parameter deviceToken: Device token data.
+  */
+  public static func setPushNotifications(deviceToken: Data!) {
```
Comment (nlutsenko) [inline Sources/Core/AppEvents/AppEventsLogger.swift]: Let's convert this into a static var, like this:
```swift
public static var pushNotificaitonsDeviceToken: Data? {
    didSet {
      FBSDKAppEvents.setPushNotificationsDeviceToken(pushNotificaitonsDeviceToken)
    }
  }
```

Also, if you could split this into a separate PR, would help improve visibility.

File: Sources/Core/AppEvents/AppEventsLogger.swift [hunk 2]
```diff
@@ -141,6 +141,18 @@ public class AppEventsLogger {
   }
 
   //--------------------------------------
+  // MARK: - Push Notification
```
Comment (nlutsenko) [inline Sources/Core/AppEvents/AppEventsLogger.swift]: Nit: `Push Notifications`

**Review by nlutsenko — CHANGES_REQUESTED** [review]
Everything looks great, except for the push notifications device token.
Can we split this into a separate PR and add the suggested fixes?
Would gladly merge it after it's done.

**Review by nlutsenko — APPROVED** [review]
Looks great. Merging as soon as Travis reports success.

**Comment by arsonik** [comment]
Great thanks !
And I've just made a PR for the push too.
